### PR TITLE
Add guard-rspec

### PR DESCRIPTION
**Pull request by timabell**
Automatically runs tests when any file changed for developer speed and convenience.

### CODE REVIEW

**Comment by blackrat** [comment]
Looks good for rspec. Do we want to add guard-rubocop to it too in order to allow it to perform all of the checks at once?

**Review by blackrat — APPROVED** [review]

File: Guardfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# More info at https://github.com/guard/guard#readme
+
+# Limit directories to watch https://github.com/guard/guard/wiki/Guardfile-DSL---Configuring-Guard#directories
+directories(%w[lib spec]) \
```
Comment (blackrat) [inline Guardfile:6]: The select should apply to the %w(lib spec) first before assigning 'directories'. Using a continuation like this makes it a bit harder to spot, but I've run it though the dsl and it throws an error

**Review by blackrat — CHANGES_REQUESTED** [review]
Error in the directories line.

File: Guardfile (hunk 1 above)
Comment (timabell) [reply Guardfile]: done, ta

**Review by blackrat — APPROVED** [review]

**Pull request merged**